apps/ledger-live-desktop/src/renderer/icons/device/interactions/NanoS/Swivel.tsx

```diff
@@ -10,7 +10,7 @@ type Props = {
   angle: number;
 };
 const NanoSSwivel = ({ angle, ...props }: Props) => {
-  const type = useTheme("colors.palette.type") as keyof typeof colors;
+  const type = useTheme().colors.palette.type as keyof typeof colors;
```

Comment:
@mcayuelas-ledger do we still need the casting part?

Reply:
Nope ! I handle it :)